test/polynomials.jl

```diff
@@ -113,8 +113,8 @@ end
     @test variables(t) == (x, y)
     @test coefficient(t) == 2
     @test exponents(t) == (2, 1)
-    @test MultivariatePolynomials.exponent(t, x) == 2
-    @test MultivariatePolynomials.exponent(t, y) == 1
+    @test degree(t, x) == 2
+    @test degree(t, y) == 1
```

Comment:
I like this change (from having degree() and exponent() to just having two methods of degree())